routes/home.js

```diff
@@ -184,7 +185,8 @@ function post_add(req, res) {
             wait_estimate: estimate,
             status: 0,
             session_id: (p.is_ta(req) || p.is_admin(req)) ? null : req.session.id,
-            topic_id: topic_id
+            topic_id: topic_id,
+            question: question
```

Comment:
Make sure to validate this field (min/max length) before storing it in the database, see above for examples.